app/src/main/java/ca/marshallasch/veil/FragmentDiscoverForums.java

```diff
@@ -6,6 +6,12 @@
 import android.content.Intent;
 import android.content.IntentFilter;
 import android.database.DataSetObserver;
+import android.graphics.Bitmap;
```

Comment:
What are the changes with the imports and the removed lines? None of those classes are used.

Reply:
removed unused imports.